### PR TITLE
all: replace `interface{}` with `any`

### LINKED ISSUE #2429: all: replace `interface{}`with `any`
### What feature would you like to be added?

For code clean-up.

### Why is this needed?

n/a

### PR DESCRIPTION
**Pull request by mymmrac**
# What issue is this addressing?
Closes #2429 

## What _type_ of issue is this addressing?
Refactoring

## What this PR does | solves

Replaces all `interface{}` to `any`, just because :)

Adds `.idea` into `.gitignore`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 *.apk
 *.tar.gz
 .vscode
+.idea
```
Comment (hajimehoshi) [inline .gitignore:7]: Do not add an unrelated change.

**Comment by hajimehoshi** [comment]
I have fixed the test failures. Please rebase your branch.

File: .gitignore (hunk 1 above)
Comment (mymmrac) [reply .gitignore:7]: I wounder, add it in separate PR or not add at all? This is JetBrains IDEs settings

Comment (hajimehoshi) [reply .gitignore]: We can add it in a separate repository. Or, you can add your own global gitignore setting

**Review by hajimehoshi — APPROVED** [review]
LGTM
